src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "../Introduce/Introduce.scss";
+import { Link } from "react-router-dom";
+
+const Introduce = () => {
+  function handleInput() {
+    console.log("input 태그동작");
+  }
+
+  function handleButton() {
+    console.log("button 태그동작");
+  }
+
+  return (
+    <div className="box">
```

Comment:
```suggestion
    <div className="introduce">
```
컴포넌트의 가장 최상위 부모태그는 컴포넌트의 이름과 동일하게 작성해주세요!